Sources/MintKit/Mint.swift

```diff
@@ -331,6 +331,9 @@ public class Mint {
         }
 
         var buildCommand = "swift build -c release"
+        for executable in executables {
+            buildCommand += " --product \(executable)"
```

Comment:
Unfortunately, this seems to only build the last executable specified (#255), so I had to make Mint run `swift build` for each executable (#256).